### DISCUSSION TITLE
Plug PDU into UPS in adjacent rack

### DISCUSSION THREAD

**Discussion by mwright77** [netbox-community/netbox#6742] [Q&A] [answered]
Is there a workaround for plugging a PDU into the a UPS which is in the next rack?  It does not show as an available device.

**Comment by candlerb** [comment, marked as answer]
You can run any cable between any racks (even in different sites).  By default, the rack of the A-side device is pre-selected, so the list of available devices for the B-side shows those in the same rack only.  You need to change the rack from the drop-down:

![image](https://user-images.githubusercontent.com/44789/125244031-bf040800-e2e6-11eb-962f-6141c22481ff.png)

**Reply by mwright77** [reply]
Thanks - that was pretty obvious, I didn't see it
